domains/sinomthamzamo.json

```
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "sinomthamzamo",
+    "email": "167751259+sinomthamzamo@users.noreply.github.com"
```

invalid email, you can safely remove this if you dont want to share your email address
```suggestion
    "username": "sinomthamzamo"
```